README.md

```diff
@@ -385,6 +385,39 @@ foreman-maintain console
 foreman-maintain config
 ```
 
+## Bash completion
+
+Bash completion is automatically installed by RPM. To use it for development setup 
+`cp ./config/foreman-maintain.completion /etc/bash_completion.d/foreman-maintain` 
+and load it to the current shell `source /etc/bash_completion.d/foreman-maintain`.
```

Comment:
When following this instructions in development, I was getting:

```
foreman-maintain-complete: command not found
```
as it assumes it's on the path. We should probably mention that the the PATH needs to be set accordingly

Reply:
We could also assume that in development, the executable will bi in `$PWD/bin`